### DISCUSSION TITLE
Anything special I need to do if I want to migrate from AIO to regular Nextcloud?

### DISCUSSION THREAD

**Discussion by pepper-lees** [nextcloud/all-in-one#2961] [Questions] [answered]
Hi,

I'm gonna be switching my server from Windows to some kinda linux distro, and in the process, I'm thinking of switching from the Docker-based all-in-one to just plain Nextcloud, [so that I can get full native speed on the Recognize app](https://github.com/nextcloud/all-in-one/discussions/2330). I see documentation on how to migrate from regular Nextcloud to AIO, but haven't found any issues/discussions/documentation on doing the reverse. Is there anything special that needs to be done in order to do that? I do have daily backups set up, so can I just set up a fresh Nextcloud instance and then restore from my backup?

Thanks!

**Comment by szaimen** [comment, marked as answer]
Hi, the easiest way in your case would probably be to follow https://docs.nextcloud.com/server/latest/admin_manual/maintenance/migrating.html. You will need the nextcloud dir (which is stored in the nextcloud_aio_nextcloud volume), a database dump (which is stored in the nextcloud_aio_database_dump volume after all containers are stopped) and the datadir (which is stored in the nextcloud_aio_nextcloud_data volume if not changed via NEXTCLOUD_DATADIR).
